src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_help.py

```diff
@@ -419,13 +419,14 @@
         - name: --max-age
           short-summary: The maximum number of seconds the client/browser should cache a preflight response.
         - name: --origins
-          short-summary: List of origin domains that will be allowed via CORS, or '*' to allow all domains.
+          short-summary: A space-separated list of origin domains that will be allowed via CORS, or '*' to allow all
+                         domains.
         - name: --methods
-          short-summary: List of HTTP methods allowed to be executed by the origin.
+          short-summary: A space-separated list of HTTP methods allowed to be executed by the origin.
         - name: --allowed-headers
-          short-summary: List of response headers allowed to be part of the cross-origin request.
+          short-summary: A space-separated list of response headers allowed to be part of the cross-origin request.
         - name: --exposed-headers
-          short-summary: List of response headers to expose to CORS clients.
+          short-summary: A space-separated list of response headers to expose to CORS clients.
```

Comment:
For consistency with the other uses in the CLI this should just say "Space-separated list of..."

Reply:
... -_- ok

Reply:
@tjprescott I'd say that @troydai's wording here is correct, and the other documentation needs to change. While this change can be temporary here, there should be a ticket for batch-editing the help.

Reply:
It wouldn't just be instances of space-separated lists.  In many, many instances in the CLI we don't lead the help text with an article. It adds no clarity, so for the sake of brevity it's often (I'd argue usually) omitted.

In this specific case, I don't know of any uses of space-separated list that use an article.